clap_derive/examples/rename_all.rs

```diff
@@ -55,7 +55,7 @@ enum Opt {
     },
 }
 
-#[derive(Clap, Debug)]
+#[derive(Subcommand, Debug)]
```

Comment:
You missed removing this.

Reply:
I haven't *missed* it, it was intentional. Actually, there are bunch of other places where `Subcommand` would be more appropriate; it just was me being a lazy butt (it was easier to keep `Clap` when it came to find-and-replace-fu).

`Subcommand` means "is going to be used with `#[clap(subcommand)]`" while `Clap` means "derive everything you can here". `Clap` derives several things, `Subcommand` is among them, but we don't really need anything except `Subcommand` here.

So, do we put `Subcommand` instead of `Clap` where needed or say "screw it" and go with God-like `Clap`? It's not super duper important anyway.

Reply:
> say "screw it" and go with God-like Clap? It's not super duper important anyway.

We should do that.